legacy-commands-v1/src/main/java/io/github/minecraftcursedlegacy/mixin/command/MixinMinecraft.java

```diff
@@ -0,0 +1,30 @@
+package io.github.minecraftcursedlegacy.mixin.command;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.Redirect;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+import io.github.minecraftcursedlegacy.api.command.DispatcherRegistry;
+import net.minecraft.client.Minecraft;
+import net.minecraft.level.Level;
+
+@Mixin(Minecraft.class)
+public class MixinMinecraft {
+	@Redirect(method = "tick()V", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/Minecraft;hasLevel()Z", ordinal = 0))
```

Comment:
this inject still uses the old method name. Should be targeted at `Lnet/minecraft/client/Minecraft;isConnectedToServer()Z`